index.html

```diff
@@ -244,11 +244,10 @@ <h3>
 concrete serializations included will be determined by the Working Group. The
 following are a non-exhaustive selection of expected input documents:
               </p>
-              <p class="input-documents"><b>Container Formats:</b>
-                <a href="https://w3c.github.io/vc-data-model/#json-web-token">VC-JSON Web Token (JWT)</a>,
-                <a href="https://w3c-ccg.github.io/data-integrity-spec/">Data Integrity</a>
+              <p class="input-documents"><b>Cryptosuites for <a href="https://w3c.github.io/vc-data-model/#json-web-token">VC-JSON Web Token (JWT)</a>:</b>
+                <a href="https://www.iana.org/assignments/jose/jose.xhtml#web-signature-encryption-algorithms">IANA JOSE Algorithms Registry</a>
```

Comment:
This is in the section where we list input documents, which are documents that the WG takes as input to work on (modify). We are not doing that w/ the IANA JOSE Algorithms Registry. [RFC7519](https://www.rfc-editor.org/rfc/rfc7519) and [RFC7515](https://www.rfc-editor.org/rfc/rfc7515), which reference the IANA JOSE Algorithms Registry as their normative registry, are already [normatively referenced](https://w3c.github.io/vc-data-model/#normative-references) in the VC Data Model specification... there is no reason to list them again here.

At best, this is an unnecessary reference (for the reasons above), at worst, it makes it sound like the VCWG is taking over control of the IANA JOSE Algorithms Registry, which we are not doing.

Reply:
vc-data-model v1 mentions these two aRFCs and not IANA JOSE Registry. I am not sure what you mean